node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
Strongly disagree here, this is no more than I a convenience over `state.value === 1`.

```js
if (led.value) {...
```

Is not improved by 

```js
if (led.isOn()) {...
```

but is shortened by 

```js
if (led.isOn) {...
```

Furthermore, it shouldn't be a function because it's not operating on any kind of input, i.e. its return value is just an assertion of its own internal state: the state of "on-ness" is a property of this LED ;)